src/Server/SSH/SSHBind.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <string>
+#include "base/types.h"
+
+struct ssh_bind_struct;
+using ssh_bind = ssh_bind_struct *;
+struct ssh_session_struct;
+using ssh_session = ssh_session_struct *;
+
+namespace ssh
+{
+
+class SSHBind
+{
+public:
+    SSHBind();
+    ~SSHBind();
+
+    SSHBind(const SSHBind &) = delete;
+    SSHBind & operator=(const SSHBind &) = delete;
+
+    SSHBind(SSHBind &&) noexcept;
+    SSHBind & operator=(SSHBind &&) noexcept;
+
+    ssh_bind get() const;
```

Comment:
same comment as for `SSHPublicKey`, is `get` needed?

Reply:
Fixed in `Remove unused get methods and fix types naming for ssh objects`